Get:27 http://security.ubuntu.com/ubuntu noble-security/multiverse amd64 Packages [28.8 kB]
Get:28 http://security.ubuntu.com/ubuntu noble-security/multiverse Translation-en [6980 B]
Get:29 http://security.ubuntu.com/ubuntu noble-security/multiverse amd64 Components [212 B]
Get:30 http://security.ubuntu.com/ubuntu noble-security/multiverse amd64 c-n-f Metadata [396 B]
Reading package lists...
E: The repository 'http://us-west-2.ec2.archive.ubuntu.com/ubuntu noble Release' no longer has a Release file.
E: The repository 'http://us-west-2.ec2.archive.ubuntu.com/ubuntu noble-updates Release' no longer has a Release file.
E: The repository 'http://us-west-2.ec2.archive.ubuntu.com/ubuntu noble-backports Release' no longer has a Release file.
Error: Process completed with exit code 100.

--- a/.github/workflows/integration-tests.yml
+++ b/.github/workflows/integration-tests.yml
@@ -49,7 +49,7 @@ jobs:
       - name: Install apt packages
         run: |
           sudo apt-get update
-          sudo apt-get install -f libcurl4-openssl-dev libgnutls28-dev httping expect libmemcached-dev
+          sudo apt-get install -y --no-install-recommends libcurl4-openssl-dev libgnutls28-dev httping expect libmemcached-dev
 
       - uses: actions/checkout@v6
       - name: Set up Python ${{ matrix.python-version }}

--- a/.github/workflows/integration-tests.yml
+++ b/.github/workflows/integration-tests.yml
@@ -48,7 +48,8 @@ jobs:
     steps:
       - name: Install apt packages
         run: |
-          sudo apt-get update && sudo apt-get install -f libcurl4-openssl-dev libssl-dev libgnutls28-dev httping expect libmemcached-dev
+          sudo apt-get update
+          sudo apt-get install -f libcurl4-openssl-dev libgnutls28-dev httping expect libmemcached-dev
 
       - uses: actions/checkout@v6
       - name: Set up Python ${{ matrix.python-version }}

--- a/.github/workflows/linter.yml
+++ b/.github/workflows/linter.yml
@@ -7,7 +7,7 @@ permissions:
 
 jobs:
   linter:
-    runs-on: blacksmith-4vcpu-ubuntu-2204
+    runs-on: blacksmith-4vcpu-ubuntu-2404
     steps:
 
       - name: Checkout branch

--- a/.github/workflows/python-package.yml
+++ b/.github/workflows/python-package.yml
@@ -54,7 +54,7 @@ jobs:
       if: startsWith(matrix.os, 'blacksmith-4vcpu-ubuntu')
       run: |
         sudo apt-get update
-        sudo apt-get install -f libcurl4-openssl-dev libgnutls28-dev httping expect libmemcached-dev
+        sudo apt-get install -y --no-install-recommends libcurl4-openssl-dev libgnutls28-dev httping expect libmemcached-dev
     - uses: actions/checkout@v6
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v6

--- a/.github/workflows/python-package.yml
+++ b/.github/workflows/python-package.yml
@@ -53,7 +53,8 @@ jobs:
     - name: Install apt packages
       if: startsWith(matrix.os, 'blacksmith-4vcpu-ubuntu')
       run: |
-        sudo apt-get update && sudo apt-get install -f libcurl4-openssl-dev libssl-dev libgnutls28-dev httping expect libmemcached-dev
+        sudo apt-get update
+        sudo apt-get install -f libcurl4-openssl-dev libgnutls28-dev httping expect libmemcached-dev
     - uses: actions/checkout@v6
     - name: Set up Python ${{ matrix.python-version }}
       uses: actions/setup-python@v6

--- a/.github/workflows/semgrep.yml
+++ b/.github/workflows/semgrep.yml
@@ -19,7 +19,7 @@ permissions:
 jobs:
   semgrep:
     name: Scan
-    runs-on: blacksmith-4vcpu-ubuntu-2204
+    runs-on: blacksmith-4vcpu-ubuntu-2404
     env:
       SEMGREP_APP_TOKEN: ${{ secrets.SEMGREP_APP_TOKEN }}
     container:

--- a/.github/workflows/smoke-tests.yml
+++ b/.github/workflows/smoke-tests.yml
@@ -32,8 +32,8 @@ jobs:
 
       - name: Install apt packages
         run: |
-          sudo apt update
-          sudo apt-get install -y procps  # Install procps to enable sysctl
+          sudo apt-get update
+          sudo apt-get install -y --no-install-recommends procps  # Install procps to enable sysctl
           sudo sysctl -w vm.overcommit_memory=1
 
       - uses: actions/checkout@v6

